frontend/src/components/Nav.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import '../App.global.scss';
+import { Link } from 'react-router-dom';
+import { Avatar } from 'wix-style-react';
+
+function Nav() {
+  const navStyle = {
+    color: '#252525',
+  };
+  const logoStyle = {
+    color: '#B38A4C',
+  };
+  const [profilePicture, setProfilePicture] = useState(null);
+
+  return (
+    <nav>
+      <h3 style={logoStyle}>Epicenter Learning Center</h3>
+      <ul className="nav-links">
+        <Link style={navStyle} to="/home">
+          <li>Home</li>
```

Comment:
Use wix-style-react components if you see it'll not take much time.
https://wix-style-react.now.sh/?path=/story/components-api-components--textbutton this seems to be it